k2/csrc/determinize.cc

```diff
@@ -17,10 +17,12 @@
  * See LICENSE for clarification regarding multiple authors
  */
 
+#include "k2/csrc/array_ops.h"
```

Comment:
this was a *very* rough draft.  we can just remove it from Makefile for now.

Reply:
revert this file